tests/conftest.py

```diff
@@ -360,3 +479,35 @@ def _load_yaml(yaml_in: str) -> PipelineConfig:
         return conf[0]
 
     return _load_yaml
+
+
+def _change_value_parameters_method_pipeline(
+    yaml_path: str,
+    method: list,
+    key: list,
+    value: list,
+    save_result: Union[None, bool] = None,
+):
+    # changes methods parameters in the given pipeline and re-save the pipeline
+    with open(yaml_path, "r") as f:
+        conf = list(yaml.load_all(f, Loader=yaml.FullLoader))
+    opened_yaml = conf[0]
+    methods_no = len(opened_yaml)
+    methods_no_correct = len(method)
+    for i in range(methods_no):
+        method_content = opened_yaml[i]
+        method_name = method_content["method"]
+        for j in range(methods_no_correct):
+            if method[j] == method_name:
+                # change something in parameters here
+                opened_yaml[i]["parameters"][key[j]] = value[j]
+                if save_result is not None:
+                    # add save_result to the list of keys
+                    opened_yaml[i]["save_result"] = save_result
+
+    with open(yaml_path, "w") as file_descriptor:
+        yaml.dump(
+            opened_yaml, file_descriptor, default_flow_style=False, sort_keys=False
+        )
+
+    return 0
```

Comment:
It looks like this function is only ever used in `test_pipeline_big.py`, and it's marked as being private in `conftest.py` - is there any reason to not put this in `test_pipeline_big.py` as a private helper function, rather than putting it in `conftest.py`?

Reply:
I was thinking that it will be used in both `test_pipeline_big` and `test_pipeline_small` if we want to extend testing a bit by modifying parameters and preview as I'm doing now in `test_pipeline_big`.

Reply:
Ok, if it's planned to be imported and used in multiple files, then that's ok, just mark it as public. At the moment, the function is implied to be private by the leading underscore in its name, and so I was confused why a private function is being imported and used elsewhere.